src/store/api.js

```diff
@@ -1,16 +1,16 @@
 export const XDAI_SUBGRAPH_URL = process.env.REACT_APP_XDAI_SUBGRAPH_URL;
 export const MAINNET_SUBGRAPH_URL = process.env.REACT_APP_MAINNET_SUBGRAPH_URL;
 export const POAP_API_URL = process.env.REACT_APP_POAP_API_URL;
+export const POAP_API_API_KEY = process.env.REACT_APP_POAP_API_API_KEY;
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
 export async function getEvents() {
-  const res = await fetch(`${POAP_API_URL}/events`)
+  const res = await fetchPOAPApi(`/events`)
   return res.json()
```

Comment:
You should remove this res.json() since you've already done it inside fetchPOAPApi